bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -848,6 +848,9 @@ private void removeMissingModuleTypes(Collection<String> moduleTypes) {
             if (rules != null) {
                 for (String rUID : rules) {
                     RuleStatus ruleStatus = getRuleStatus(rUID);
+                    if (ruleStatus == null) {
```

Comment:
Yes agree, but this was only added to stop debuger here. We must not have a situation to have ruleStatus == null here, because all added rule must have a status. I'll remove it.